diff_settings.py

```diff
@@ -115,5 +115,16 @@ def apply(config, args):
             apply_psx_base(config, version, name)
         else:
             apply_psx_bin(config, version, name)
+
+        # HACK: allow to watch for the 'src/*_psp' directory
+        if version.startswith("psp"):
+            source_directories = config["source_directories"]
+            new_source_directories = []
+            for source_dir in source_directories:
+                new_source_directories.append(source_dir)
+                if source_dir.startswith("src/"):
+                    new_source_directories.append(source_dir + "_psp")
+            config["source_directories"] = new_source_directories
```

Comment:
This could be simplified to:
```
        if version.startswith("psp") and config["source_directories"]:
            add_dirs = [f"{dir}_psp" for dir in config["source_directories"] if dir.startswith("src/")]
            config["source_directories"].extend(add_dirs)
```

This is assuming config["source_directories"] can be assumed to be a list.